index.html

```diff
@@ -188,75 +188,115 @@ <h4 class="colorh4">CANOPY</h4>
+                        <i class="fa fa-twitter social"></i>
+                    </a>
                 </li>
-                <li class="headlang">
-                    <h5>Frameworks</h5>
-                    <i class="fa fa-angle-down fontarrows"></i>
+                <li>
+                    <a title="The link will take you to my instagram profile"
+                        href="https://www.instagram.com/mpdaddympd/">
+                        <i class="fa fa-instagram social"></i>
+                    </a>
                 </li>
-                <li class="headlang">
-                    <h5>Skills</h5>
-                    <i class="fa fa-angle-down fontarrows"></i>
+                <li>
+                    <a title="The link will take you to my linkin profile"
+                        href="https://www.linkedin.com/in/anold-mwawasi">
+                        <i class="fa fa-linkedin social"></i>
+                    </a>
                 </li>
             </ul>
+            <div class="getresume">
+                <a title="get my resume" href="#">Get my resume</a>
+            </div>
         </article>
-    </section>
+        <article id="what_i_do" class="padding">
 
+          <ul class="gridcolumn">
+            <li>
+                 <div class="mygrid">
+                    <h5>Languages</h5>
+                    <i class="fa fa-angle-down fontarrows right"></i>
+                 </div>
+                 <ul class="langcontainer">
+                    <li class="langgrid">
+                        <img class="marginleft" src="images/Ellipse 1.png" alt="">
+                        <p>javascript</p>
+                    </li>
+                    <li class="langgrid">
+                        <img src="images/Ellipse 1 (1).png" alt="">
+                        <p>HTML</p>
+                    </li>
+                    <li class="langgrid">
+                        <img src="images/Ellipse 1 (2).png" alt="">
+                        <p>CSS</p>
+                    </li>
+                 </ul>
+            </li>
+            <li class="headlang">
+              <div class="mygrid">
+                <h5>Frameworks</h5>
+                <i class="fa fa-angle-right fontarrows"></i>
+              </div>
+            </li>
+            <li class="headlang">
+               <div class="mygrid">
+                <h5>Skills</h5>
+                <i class="fa fa-angle-right fontarrows"></i>
+               </div>
+            </li>
+          </ul>
+        </article>
+
+<!---contact me article-->
+
+        <footer>
+            <section class="footer">
+              <h2>Contact me</h2>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I'd love to help with it.
+              </p>
+              <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Kindly makes sure that the form are  submitted successfully with valid input values. When I try to submit the form with values I got the following error related with the `{form_id}`, So to fix the issues Kindly use your own `{form_id}` by creating in the [formspree service](https://formspree.io/html).
![Capture](https://github.com/MPDADDY/portfolio/assets/93598189/0ccf4ac5-95a9-4976-a202-488b326da274)